openlibrary/plugins/openlibrary/js/availability.js

```diff
@@ -51,9 +51,6 @@ function initAvailability() {
         '^/account/books/[^/]+': { // readinglog
             filter: false
         },
```

Comment:
Do reading logs still need this?

Reply:
Yes, this PR fixes realtime availability checks for reading log. I tried to use the canonical `add_availability` on the backend without success.